drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -77,5 +87,14 @@ def resolve(input:)
         errors: errors,
       }
     end
+
+    private def perform_side_effects(record)
+      GrdaWarehouse::Tasks::IdentifyDuplicates.new.delay.run! if record.is_a? Hmis::Hud::Client
```

Comment:
Should we run this when a client is updated, or only when they are created?

Reply:
If Name, DOB, or SSN (or MCI ID) change, we should probably re-run this, but it's also ok to leave it to the nightly process to catch those changes.
We could run `GrdaWarehouse::Tasks::IdentifyDuplicates.new.delay.match_existing!` on update instead of the full thing, which should be faster.

Reply:
Updated with https://github.com/greenriver/hmis-warehouse/pull/2869/commits/3f1beec880161349571da7daed34636932188c42 to run `run!` for new record and `match_existing!` for existing records. Should be simple to change what we run based on what fields have changed, using active model Dirty. I'll just wait on that until we have specific specs (around MCI id for example).